.idea/.gitignore

```diff
@@ -0,0 +1,10 @@
+# Default ignored files
```

Comment:
delete the `.idea` folder and files in it

Reply:
k